content/docs/examples/userauthentication.md

```diff
@@ -80,45 +104,46 @@ We need one more value from Auth0: **the `issuer_url` for your Auth0 account**.
 ### Creating our Authorizer
 We can now create an `Authorizer` for our Cdev API. 
```

Comment:
We should add some more detail on what an `Authorizer` is. I can help write that if needed.

Reply:
Added a simple definition and tool tip containing links